tools/tools.go

```diff
@@ -86,6 +90,42 @@ func Sed(oldValue, newValue, filePath string) error {
 	return err
 }
 
+func SshRun(userName, userPassword, server, cmd string) (string, error) {
```

Comment:
Care to port over also the SSH Copy one we have since at it? https://github.com/rancher-sandbox/cOS-toolkit/blob/master/tests/sut/sut.go#L266

Reply:
also, small nit: better uppercase for ssh, like `SSHRun`, or either `RunSSH`

Reply:
agree, not default ssh port support, would be welcome, quite often it is changed from default fro security reasons

Reply:
> Care to port over also the SSH Copy since at it?

Ok, on it.

> also, small nit: better uppercase for ssh, like SSHRun, or either RunSSH

Done in a next commit.